lib/Server.js

```diff
@@ -58,6 +59,8 @@ class Server {
     this.compiler = compiler;
     this.options = options;
 
+    handleStdin(options);
```

Comment:
hm, why we need it here?

Reply:
@evilebottnawi Do you think no helper would be better? I just thought having helper would be cleaner

Reply:
i think we need this inside server file, it is logic for CLI only